.github/workflows/test-performance.yml

```
@@ -0,0 +1,96 @@
+          echo "Failed to populate kubeconfig"
+          exit 1
+        fi
+
+        if ! kubelogin convert-kubeconfig -l azurecli; then
+          echo "Failed to convert kubeconfig"
+          exit 1
+        fi
+    - name: Setup k6
+      uses: grafana/setup-k6-action@v1
+    - name: Run K6 tests (${{ inputs.testSuitePath }})
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:4:108: Double quote to prevent globbing and word splitting

(shellcheck)